slack-base/src/main/java/com/hubspot/slack/client/models/interaction/BlocksLoadOptionsResponseIF.java

```diff
@@ -19,6 +20,13 @@
 @HubSpotStyle
 @JsonNaming(SnakeCaseStrategy.class)
 public interface BlocksLoadOptionsResponseIF extends BaseSlackOptionsResponse {
+  String TYPE = "blocks_load_options_response";
+
+  @Derived
+  default String getType() {
+    return TYPE;
+  }
```

Comment:
Why do we need this?

Reply:
Looks like we didn't. I thought it is used by Jackson to determine type but looks like I was wrong. Removing this.